react/javascript/demo-embeds/src/components/DashboardEmbed/DashboardEmbed.jsx

```diff
@@ -112,6 +112,7 @@ export const DashboardEmbed = ({ embedType }) => {
           if (hostUrl) {
             LookerEmbedSDK.init(hostUrl)
             const embed = LookerEmbedSDK.createDashboardWithId(dashboardId)
+              .withNext('-legacy')
```

Comment:
Maybe some of the other references, like the component name, should be updated to reflect the legacy status.

Reply:
Makes sense. Done